src/static/riot/competitions/tile/competition_tile.tag

```diff
@@ -1,104 +1,135 @@
-                    <!--<img src="{ logo }">-->
-                </div>
+    <a class="link-no-deco" href="https://google.com">
+        <div class="tile-wrapper">
+            <div class="ui square tiny bordered image img-wrapper">
+                <img src="{logo}">
             </div>
-            <div class="nine wide column">
-                <div class="ui row">
-                    <div class="twelve wide column">
-                        <h1 class="ui large header">
-                            {title}
-                        </h1>
-                        <p class="comp-tile-paragraph">
-                            A competition description
-                            {description}
-                        </p>
-                    </div>
-                    <div class="four wide right justify left align column">
-                        <div align="right" class="content">
-                            <i>Organized by: <b>Someone</b></i>
-                        </div>
-                    </div>
-                </div>
-                <div class="ui row">
-                    <div class="sixteen wide column">
-                        <div class="content">
-                            <!--Tags?-->
-                        </div>
-                    </div>
-                </div>
-                <div class="ui row">
-                    <div class="sixteen wide column">
-                        <div class="content">
-                            Tags: <b>Beginner</b>, <b>AutoML</b>
-                            <br>
-                            Admins: <b>tthomas63</b>
-                        </div>
-                    </div>
-                </div>
+            <div class="comp-info">
+                <h4 class="heading">
+                    {title}
+                </h4>
+                <p class="comp-description">
+                    {description}
+                </p>
+                <p class="organizer">
+                    <em>Organized by: <strong>{created_by}</strong></em>
+                </p>
             </div>
-            <div class="three wide blue column center aligned comp-tile-full-height">
-                <i>Comp deadline:</i>
-                <i>August 14 2018</i>
-                <div class="ui divider"></div>
-                <i>Phase deadline:</i>
-                <i>April 14 2018</i>
+            <div class="comp-stats">
+                {pretty_date(created_when)}
                 <div class="ui divider"></div>
-                <i>17 participants</i>
+                <strong>{participant_count}</strong> Participants
             </div>
         </div>
-    </div>
+    </a>
 
     <script>
+        self.pretty_date = function (date_string) {
+            if (!!date_string) {
+                return luxon.DateTime.fromISO(date_string).toLocaleString(luxon.DateTime.DATE_FULL)
+            } else {
+                return ''
+            }
+        }
+
+        $(document).ready(function () {
```

Comment:
Making use of Riot, instead of `$(document).ready()` we can use:
```javascript
let self = this

self.on('mount', () => {

}
```

Reply:
Swapped to riot mount